reporting-batch/src/main/java/it/gov/pagopa/reporting/service/FlowsService.java

```diff
@@ -40,8 +40,37 @@ public FlowsService(String storageConnectionString, String flowsTable, String fl
         this.logger = logger;
     }
 
+//    private void createTable() throws URISyntaxException, InvalidKeyException, StorageException {
```

Comment:
it is ok, but commented code is a code smell for sonar

Reply:
you're right, in fact this is useless code that is only for local development for azurite